test/print.jl

```diff
@@ -793,3 +793,132 @@ end
     @variable(model, z, base_name = "z^1")
     io_test(IJuliaMode, z, "z\\^1")
 end
+
+@testset "Print solution summary" begin
+
+    m = Model()
```

Comment:
Nit: By convention we use `model` instead of `m`. It's confusing to have both `m` and `model` (looks like the latter isn't actually used).

Reply:
I fixed.